dev-infra/typescript/tsconfig.json

```diff
@@ -50,7 +50,7 @@
         "noStrictGenericChecks": false,
         "preserveConstEnums": false,
         "noUnusedParameters": true,
-        "noUnusedLocals": true,
+        "noUnusedLocals": false,
```

Comment:
sorry, it's by design

Reply:
in your project

`tsconfig.json`

```json5
{
 "extends": "@angular-ru/typescript/tsconfig.json",
 "compilerOptions": {
    // ...
  }
}
```

and then for `tsconfig.spec.json`

```json5
{
 "extends": "./tsconfig.json",
 "compilerOptions": {
     // ...
    "noUnusedLocals": false
  }
}
```

Reply:
You can just add this example to README.md

Reply:
even if the same errors are covered by eslint? 

the reason we made this is that webpack refuses to compile the project till you clean the unused variables. sometimes it takes a lot of time: you need to clean the variable, then clean another variables related to it, then clean the imports. and all this is just because you wanted to check how it works without variable.

if not, then ok. we set it inside our project.

Reply:
> we'll set it inside our project.

Yes, this is what you should do